### PR TITLE
Edit profile

### PR DESCRIPTION
**Pull request by mahsa-kh**
-added edit profile item to action items on navbar
-added edit function

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,12 @@
 class ApplicationController < ActionController::Base
+  before_action :authenticate_user!
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  def configure_permitted_parameters
+    # For additional fields in app/views/devise/registrations/new.html.erb
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:first_name, :last_name, :location])
+
+    # For additional in app/views/devise/registrations/edit.html.erb
+    devise_parameter_sanitizer.permit(:account_update, keys: [:username])
```
Comment (danielppolo) [inline app/controllers/application_controller.rb:10]: You should change the `username` for the actual attributes from our User. 
(first_name, last_name, location)

Comment (mahsa-kh) [reply app/controllers/application_controller.rb:10]: Shall I keep the :username name or change the whole part to:
devise_parameter_sanitizer.permit(:account_update, keys: [:first_name, :last_name, :location])

Comment (danielppolo) [reply app/controllers/application_controller.rb]: Like the second, we don't have any username in our User

Comment (mahsa-kh) [reply app/controllers/application_controller.rb]: ok

**Comment by mahsa-kh** [comment]
Edited Application Controller params

**Pull request merged**